ffsb_stats.c

```diff
@@ -231,6 +231,7 @@ void ffsb_statsd_print(ffsb_statsd_t *fsd)
 		}
 		overall_calls += fsd->num_values[i];
 	}
+#if 0
```

Comment:
Is this printing causing test failure every time?
If yes, then how it was passing earlier?
If no, then why some recent failures observed?
If test failure is intermittent then change looks ok to me.

Reply:
> Is this printing causing test failure every time? 

Not every time, it depends on how the nodes and network.

> If yes, then how it was passing earlier? If no, then why some recent failures observed? If test failure is intermittent then change looks ok to me.

Just recently the failure are more often. The teuthology test are running in the test nodes somewhere I don't know what has happened there, maybe the network has changed, or something else. Locally my test itself finished very fast and almost the same with the teuthology tests, but after that the printing is much faster than in teuthology test nodes.

Reply:
I think this makes sense. From https://pulpito.ceph.com/vshankar-2022-04-21_04:56:30-fs-wip-vshankar-testing1-20220419-121051-testing-default-smithi/6798826/

```
2022-04-21T05:35:41.825 INFO:tasks.workunit.client.0.smithi025.stdout:
2022-04-21T05:35:41.826 INFO:tasks.workunit.client.0.smithi025.stdout:Discrete overall System Call Latency statistics in millisecs
2022-04-21T05:35:41.826 INFO:tasks.workunit.client.0.smithi025.stdout:=====
2022-04-21T05:35:41.826 INFO:tasks.workunit.client.0.smithi025.stdout:
2022-04-21T05:35:41.826 INFO:tasks.workunit.client.0.smithi025.stdout:Overall Calls: 135035334
2022-04-21T05:35:41.826 INFO:tasks.workunit.client.0.smithi025.stdout:=====
2022-04-21T05:35:41.827 INFO:tasks.workunit.client.0.smithi025.stdout:Values[ms]:
2022-04-21T05:35:41.827 INFO:tasks.workunit.client.0.smithi025.stdout:====
2022-04-21T05:35:41.827 INFO:tasks.workunit.client.0.smithi025.stdout:[   open] Total calls:        52707
....
....
....
2022-04-21T08:30:06.341 INFO:tasks.workunit.client.0.smithi025.stdout:0.010000
2022-04-21T08:30:06.343 INFO:tasks.workunit.client.0.smithi025.stdout:0.010000
2022-04-21T08:30:06.343 INFO:tasks.workunit.client.0.smithi025.stdout:0.010000
2022-04-21T08:30:06.345 INFO:tasks.workunit.client.0.smithi025.stdout:0.011000
2022-04-21T08:30:06.345 INFO:tasks.workunit.client.0.smithi025.stdout:0.011000
2022-04-21T08:30:06.345 INFO:tasks.workunit.client.0.smithi025.stdout:0.011000


```